libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/usecase/FlowUseCase.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 Google LLC
```

Comment:
I don't think we should be adding Google's source code to our SDK so casually. @LukasPaczos @zugaldia  does this need a more formal review?

Reply:
Apache is a permissive license which allows using code commercially as well as altering the code. Having said that, I'd be interested in learning more why we need to bring in this code here. I'll follow up with @tomaszrybakiewicz directly to understand. Let's not merge this PR until I have an opportunity to do that.

Reply:
Don't we have to disclose the usage in our own license.md file? That gets difficult to track when it's source code in our SDK vs. a binary that we're importing.

Reply:
Since we are not using UseCase approach, I am removing these files. This is no longer valid.